Doc/c-api/tuple.rst

```diff
@@ -56,8 +56,10 @@ Tuple Objects
 
 .. c:function:: PyObject* PyTuple_GetItem(PyObject *p, Py_ssize_t pos)
 
-   Return the object at position *pos* in the tuple pointed to by *p*.  If *pos* is
-   out of bounds, return ``NULL`` and set an :exc:`IndexError` exception.
```

Comment:
You can say:

If pos is negative or out of bound, ...

To document that IndexError is raised if pos is negative.